plugins/InspectionGadgets/InspectionGadgetsAnalysis/resources/messages/InspectionGadgetsBundle.properties

```diff
@@ -2373,6 +2374,7 @@ unnecessary.fully.qualified.name.fix.family.name=Replace fully qualified name
 return.of.collection.field.fix.family.name=Make return collection 'unmodifiable'
 remove.redundant.substring.fix.family.name=Remove redundant 'substring()' call
 remove.redundant.substring.fix.text=Use ''{0}'' and remove redundant ''substring()'' call
+remove.redundant.change.case.fix.text=Use ''equalsIgnoreCase()'' and remove redundant ''{0}'' call
```

Comment:
I suggest generalizing existing message instead: `Use ''{0}()'' and remove redundant ''{1}()'' call` and reuse it in both places. This would reduce the amount of work for our translators.

Reply:
I changed to "Use ''{0}()'' and remove redundant ''{1}()'' call"